### ISSUE TITLE
Connected and Unconnected Items Have Identical Screen Reader Responses

### ISSUE THREAD

**Issue by vperezcadavid99** [phetsims/circuit-construction-kit-dc#231]
**Test device**
Asus Zenbook\

**Operating System**
Win 11

**Browser**
Firefox + NVDA

**Problem description**
From https://github.com/phetsims/qa/issues/1352

In this example, I noticed that when focusing on a wire connected to another item (such as a battery), the screen reader announces “Wire 1 of 1, movable.” However, after adding another wire that is not connected to any item and focusing on it, the screen reader announces the same message: “Wire 1 of 1, movable.” I think these two cases should have different responses, since the wires are in different states (connected vs. unconnected). More generally, there is no screen reader description indicating whether an item is connected or unconnected.

This is shown in the a11y view, but not implemented for the screen reader

<img width="2667" height="954" alt="Image" src="https://github.com/user-attachments/assets/34856fa5-0ea8-4687-92b7-8d71ee680f5c" />

**Visuals**

https://github.com/user-attachments/assets/a48c13fb-d0f0-4297-b8fd-92730230c3e1

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Circuit Construction Kit: DC‬
URL: https://phet-dev.colorado.edu/html/circuit-construction-kit-dc/1.5.0-dev.3/phet/circuit-construction-kit-dc_all_phet.html
Version: 1.5.0-dev.3 2026-01-30 16:08:59 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/144.0.0.0 Safari/537.36
Language: en-US
Window: 1440x731
Pixel Ratio: 2/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}
<details/>

**Comment by arouinfar** [comment]
> I think these two cases should have different responses, since the wires are in different states (connected vs. unconnected). More generally, there is no screen reader description indicating whether an item is connected or unconnected.

To get the full context, users need to use their cursor to read the state description. Tabbing will only provide part of the picture.  This is true of most sims, not just CCK. 

We offer context about the group structure in a few places, including the current details of the Screen Summary, the accessible paragraph describing the Construction Area, heading names, and the accessible names of the first/last item in a group.

Within the context of a group, it makes more sense to have Wires 1, 2, 3... If we used a global index, a circuit could contain Wires 2, 4, and 7, which makes it harder to understand how the components are connected.

That said, this is the sort of thing we should be on the lookout for in interviews, so I'll tag https://github.com/phetsims/circuit-construction-kit-common/issues/1205 and close this issue.
